app/src/main/java/com/fastlink/zekrapp/appData/data/zekrdata.kt

```diff
@@ -1,4 +1,4 @@
-package com.fastlink.zekrapp.data.data
+package com.fastlink.zekrapp.appData.data
```

Comment:
"appData" is not better than "data", make the package data then inside it repository. Or just put everything inside one package called "data". 

Take a look at this:
https://github.com/android/compose-samples/tree/main/JetLagged/app/src/main/java/com/example/jetlagged/data

Reply:
Thank you for providing this resource; I appreciate it. Please note that I will delete this folder at the end of the project. Initially, we will retrieve the data from a local database, but I included this folder to facilitate the UI implementation. Kak Ahmad instructed me to focus solely on creating the UI and not to handle the data until the roadmap is completed. While this is not ideal, I will make an effort to manage it. For the time being, I am considering using a CSV file as an alternative to the data currently stored in the ArrayList

Reply:
I'm only talking about the package names.